app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,50 +1,51 @@
-            android:paddingBottom="50dp">
+        android:id="@+id/homeUrlBar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:paddingBottom="50dp"
+        android:paddingLeft="12dp"
+        android:paddingRight="12dp">
 
         <!-- TODO: analyze the params below -->
-        x<org.mozilla.focus.widget.InlineAutocompleteEditText
-                android:id="@+id/urlInputView"
-                android:layout_width="0dp"
-                android:layout_height="40dp"
-                android:layout_weight="1"
-                android:layout_gravity="center_horizontal"
-                android:hint="@string/urlbar_hint"
-                android:imeOptions="actionGo|flagNoExtractUi|flagNoFullscreen"
-                android:background="@color/homeUrlBarBackground"
-                android:inputType="textUri"
-                android:lines="1"
-                android:layout_marginEnd="20dp"
-                android:paddingTop="8dp"
-                android:paddingBottom="8dp"
-                android:paddingStart="8dp"
-                android:paddingEnd="8dp"
-                android:requiresFadingEdge="horizontal"
-                android:selectAllOnFocus="true"
-                android:textColor="#ffffff"
-                android:textColorHighlight="@color/colorAccent"
-                android:textColorHint="@color/colorTextInactive"
-                android:textSize="15sp"
-                android:textCursorDrawable="@drawable/cursor"
-                android:importantForAutofill="no"
-                tools:targetApi="o" /><!-- importantForAutofill is only supported on Android O+ -->
+
+        <org.mozilla.focus.widget.InlineAutocompleteEditText
+            android:id="@+id/urlInputView"
+            android:layout_width="0dp"
+            android:layout_height="40dp"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginEnd="20dp"
+            android:layout_weight="1"
+            android:background="@color/homeUrlBarBackground"
+            android:hint="@string/urlbar_hint"
+            android:imeOptions="actionGo|flagNoExtractUi|flagNoFullscreen"
+            android:importantForAutofill="no"
+            android:inputType="textUri"
+            android:lines="1"
+            android:paddingBottom="8dp"
+            android:paddingEnd="8dp"
+            android:paddingStart="8dp"
+            android:paddingTop="8dp"
+            android:requiresFadingEdge="horizontal"
+            android:selectAllOnFocus="true"
+            android:textColor="#ffffff"
+            android:textColorHighlight="@color/colorAccent"
+            android:textColorHint="@color/colorTextInactive"
+            android:textCursorDrawable="@drawable/cursor"
+            android:textSize="15sp"
+            tools:targetApi="o" /><!-- importantForAutofill is only supported on Android O+ -->
```

Comment:
I'm going to assume you didn't change this InlineAutocompleteEditText bc this is too many items to compare by hand.